__tests__/app.test.js

```diff
@@ -164,3 +166,23 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("returns an array of topic objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length > 0).toBe(true);
```

Comment:
is there a jest matcher we can use for this to make our tests a bit more readable?

Reply:
Thanks, will look into them and adjust it!